polling_stations/apps/data_collection/management/commands/import_rochdale.py

```diff
@@ -49,22 +43,28 @@ def address_record_to_dict(self, record):
         if uprn == "23081457":
             rec["postcode"] = "OL12 9AA"
 
-        if record.addressline6.strip() == "OL16 3BX":
-            return None
-
-        if record.addressline6.strip() == "M24 5TG":
-            return None
-
         if uprn == "23099282":
             rec["postcode"] = "OL11 3AU"
 
-        if record.addressline6.strip() == "OL16 5SJ":
-            return None
-
-        if record.addressline6.strip() == "M24 2YS":
-            return None
-
         if uprn in ["23095362", "23095226"]:
             rec["postcode"] = "OL12 9NE"
 
+        # Not quite right, safer to throw away
+        if record.addressline6.strip() in [
+            "OL16 3BX",
+            "OL16 5SJ",
+            "M24 2YS",
+            "M245TG",
+            "M245TN",
```

Comment:
I don't think these last 2 are actually going to match anything. Usually the postcodes in `record.addressline6` (the input file) will have a space in them.

Reply:
...and again!